ludwig/backend/ray.py

```diff
@@ -470,16 +459,12 @@ def tune_batch_size(
             )
         )
 
-    def tune_learning_rate(self, config, training_set: RayDataset, **kwargs) -> float:
+    def tune_learning_rate(self, config, **kwargs) -> float:
```

Comment:
No need to run this as a `ray.remote` function anymore, since it's not doing any actual training. Maybe also rename from `tune_learning_rate` to `detect_learning_rate`.